org.freecadweb.FreeCAD.appdata.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.freecadweb.FreeCAD</id>
+  <project_license>LGPL-2.1</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <developer_name>The FreeCAD Team</developer_name>
+  <name>FreeCAD</name>
+  <summary>An open source parametric 3D CAD modeler</summary>
+  <description>
+    <p>
+      FreeCAD is a parametric 3D modeler. Parametric modeling
+      allows you to easily modify your design by going back into
+      your model history and changing its parameters. FreeCAD is
+      open source (LGPL license) and completely modular, allowing
+      for very advanced extension and customization.
+    </p>
+    <p>
+      FreeCAD is multiplatfom, and reads and writes many open
+      file formats such as STEP, IGES, STL and others.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://www.freecadweb.org/wiki/images/2/2d/FreeCAD011.png</image>
+    </screenshot>
+  </screenshots>
+  <url type="homepage">https://www.freecadweb.org</url>
+  <url type="bugtracker">https://www.freecadweb.org/tracker</url>
+  <url type="help">https://forum.freecadweb.org</url>
+  <url type="donation">https://www.freecadweb.org/wiki/Donate</url>
+  <update_contact>yorik_AT_uncreated.net</update_contact>
+  <content_rating type="oars-1.1" />
+  <releases>
+    <release version="0.17" date="2018-04-06"></release>
+  </releases>
+</component>
```

Comment:
Shouldn't it also have `launchable` tag?

Reply:
This is add by `appstream-compose`.

Reply:
It is added because you didn't.

Reply:
We also tell people not to add categories, because they are added automatically (from the desktop file), so why should this be different?

Reply:
Categories would be tedious duplication, while specifying launchable explicitly is done on single place and just a good practice.